GetBPDCitizen/handler.ts

```diff
@@ -170,8 +170,9 @@ export function GetBPDCitizen(
     (context, { user, fiscalCode, citizenIdType }) => ({
       AuthLevel: isAdminAuthLevel(user, adb2cAdminGroup) ? "Admin" : "Support",
       Citizen: fiscalCode,
+      Email: user.emails[0],
       OperationName: "GetBPDCitizen",
-      PartitionKey: user.oid, // Can we use email?
+      PartitionKey: user.oid,
```

Comment:
can we retrieve user oids giving the user email? (you have to specify the partitionkey when searching in table storage). otherwise consider using the email as partition key

Reply:
The reason why we are using the user `oid` is to unify the history of a user account because the email could be modified. The `oid` value can be found inside ADB2C user detail page and used to perform a query.
cc. @pp-ps